src/components/ec-checkbox/ec-checkbox.vue

```diff
@@ -114,48 +119,52 @@ export default {
       type: Boolean,
     },
   },
-  emits: ['update:modelValue'],
-  data() {
-    return {
-      uid: getUid(),
-      inputIsFocused: false,
-    };
-  },
-  computed: {
-    id() {
-      return `ec-checkbox-${this.uid}`;
-    },
-    errorId() {
-      return this.isInvalid ? `ec-checkbox-error-${this.uid}` : null;
-    },
-    isInvalid() {
-      return !!this.errorMessage || !!this.$slots['error-message'];
-    },
-    inputModel: {
-      get() {
-        return this.modelValue;
-      },
-      set(checked) {
-        this.$emit('update:modelValue', checked);
-      },
-    },
-  },
-  watch: {
-    indeterminate(newValue) {
-      this.updateIndeterminate(newValue);
-    },
+);
+
+const uid = ref(getUid());
```

Comment:
I'd make this one static (no `ref`). We don't expect this one to change at runtime. The id is given when the component is created and that's it. It's less work for vue's reactivity and we don't have to use `uid.value` everywhere.